m4_sessions/automation/session_one/eight_mike_justin_jesse_ethan_dan.md

```diff
@@ -1,3 +1,53 @@
+
+## Pre-Session
+
+
+### Survey Questions
+Survey to be sent either prior to session or immediately after ( prior may lead to better response? )
+
+#### Survey to lighten the mood
+  * funny questions like:
+   1. When the robots attack, what’s the first thing you do?
+   2. If you could have one process in your life automated tomorrow what would it be?
+   3. What automated process today do you appreciate the most
+   4. Write your apology to the person whose job you stole by automating that thing.
+   
+### Pre-Reading
+
+Send pre-reading out at least 24 hours before the session
+
+*   ["Machines for Thinking"](http://www.economist.com/news/books-and-arts/21669597-computers-will-get-smarter-humans-charge-machines-thinking) (_The Economist_: October 1, 2015)
+*   "The Rise of the Robots by Martin Ford / Humans Need Not Apply by Jerry Kaplan – review"
+https://www.theguardian.com/books/2015/oct/01/the-rise-of-robots-humans-need-not-apply-review (_The Guardian_: October 1, 2015)
+*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+
+### Groupings
+  We will group members by opposing stances. Have students sit in a spectrum of how much they like the idea of automation, and from that hand select one each from opposing sides and a middle. This should facilitate the most conversation and opposing viewpoints.
+  
+ ## Session Outline
+
+| Timing        | Facilitator Talking Points           | Participant/Facilitator Actions  |
+| ------------- | ------------- | ----- |
+| 5 min | Introduction: Welcome participants.  Introduce yourself and your role.  Review the outcomes for the session and the connection to the greater Turing mission.  Ask students about survey, and read out some responses.
+| 5 min | Warm Up: Kick off an activity where students will organize in the room along a spectrum of their attitudes toward automation. Describe one end as representing people who feel strongly that automation is, as Elon Musk says, "summoning a demon." On the other end, people who feel automation is going to take humanity to new, positive, exciting potential. | Facilitator will instruct students to line up in between these poles as appropriate. At facilitator's prompting, students should line up along the spectrum of attitudes toward automation. This arrangement will be used to select dynamic groups |
+| 2 min | Introduce video and reflection questions to students. <br> 1) How serious an impact do you think automation will have on our economy from now until you retire? <br> 2) Where do you think you fall across the spectrum of people that will be affected by these changes? <br> 3) What do you think it means to be a responsible citizen around this issue? <br> 4) What role can Turing play in advancing civic responsibility in response to the challenges automation will bring?     
```

Comment:
do you think it'd be helpful to have these questions written on the board during the viewing of each video so that they'll be reminded of what they should be thinking about?